cdap-integration-test/src/main/java/co/cask/cdap/test/IntegrationTestBase.java

```diff
@@ -312,13 +339,41 @@ protected ApplicationManager deployApplication(Id.Namespace namespace,
   }
 
   protected ApplicationManager deployApplication(Class<? extends Application> applicationClz) throws IOException {
-    return deployApplication(Id.Namespace.DEFAULT, applicationClz);
+    return deployApplication(getConfiguredNamespace().toId(), applicationClz);
   }
 
   protected ApplicationManager getApplicationManager(ApplicationId applicationId) throws Exception {
     return getTestManager().getApplicationManager(applicationId);
   }
 
+  private void doClear(NamespaceId namespace) throws Exception {
```

Comment:
yea, I didn't think this was worthy of a backend change, with just this use case.